src/guidellm/__main__.py

```diff
@@ -11,7 +11,7 @@
 Example:
 ::
     # Run a benchmark against a model
-    guidellm benchmark run --target http://localhost:8000 --data dataset.json \\
```

Comment:
I won't block on this, but pointing users towards the full command would make --help more helpful.